src/traits/LazyDataLoadTrait.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace luya\admin\traits;
+
+trait LazyDataLoadTrait
+{
+    /**
+     * @param array|\Closure $data
+     * @return array|\Traversable
+     */
+    protected function loadData($data)
```

Comment:
maybe we could use a more generic name, since this is used as trait it might conflict quickly. what about `lazyLoadData`? Please add some more phpdocs data to the param, maybe an example of what is possible. also a description is missing.